src/main/java/bio/terra/common/SynapseColumn.java

```diff
@@ -62,24 +62,17 @@ public static String translateDataType(TableDataType datatype, boolean isArrayOf
         return "varbinary";
       case DATE:
         return "date";
-      case DATETIME:
-      case TIMESTAMP:
+      case DATETIME, TIMESTAMP:
```

Comment:
maybe use an expression switch, and put the `return` outside the switch

Reply:
good call, will do